fastlane/lib/fastlane/actions/commit_github_file.rb

```diff
@@ -0,0 +1,184 @@
+                                       description: "The server url. e.g. 'https://your.internal.github.host/api/v3' (Default: 'https://api.github.com')",
+                                       default_value: "https://api.github.com",
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Please include the protocol in the server url, e.g. https://your.github.server/api/v3") unless value.include? "//"
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :api_token,
+                                       env_name: "FL_COMMIT_GITHUB_FILE_API_TOKEN",
+                                       description: "Personal API Token for GitHub - generate one at https://github.com/settings/tokens",
+                                       sensitive: true,
+                                       is_string: true,
+                                       default_value: ENV["GITHUB_API_TOKEN"],
+                                       optional: false),
+          FastlaneCore::ConfigItem.new(key: :branch,
+                                       env_name: "FL_COMMIT_GITHUB_FILE_BRANCH",
+                                       description: "The branch that the file should be committed on (default: master)",
+                                       default_value: 'master',
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :path,
+                                       env_name: 'FL_COMMIT_GITHUB_FILE_PATH',
+                                       description: 'The relative path to your file from project root e.g. assets/my_app.xcarchive',
+                                       optional: false,
+                                       is_string: true,
+                                       verify_block: proc do |value|
+                                         value = File.expand_path(value)
+                                         UI.user_error!("File not found at path '#{value}'") unless File.exist?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :message,
+                                       env_name: "FL_COMMIT_GITHUB_FILE_MESSAGE",
+                                       description: "The commit message. Defaults to the file name",
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :secure,
+                                     env_name: "FL_COMMIT_GITHUB_FILE_SECURE",
+                                     description: "Optionally disable secure requests (ssl_verify_peer)",
+                                     is_string: false,
+                                     default_value: true,
+                                     optional: true),
+          FastlaneCore::ConfigItem.new(key: :debug,
```

Comment:
Would be great to not include this option and instead use the `UI.verbose` method that we have. I think they are serving the same kind of purpose?